woocommerce/woocommerce
# Add new test scenario for mini-cart inclusive tax price

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes # [699](https://github.com/woocommerce/woocommerce-quality/issues/699) in Woo Quality

Also improved quality of the test itself along with those changes of a new added scenario.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Checkout this branch locally
2. Execute the test: `pnpm test:e2e-pw ./tests/e2e-pw/tests/shopper/mini-cart.spec.js`
4. Ensure it is passing



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
